flang/lib/Common/CMakeLists.txt

```diff
@@ -34,8 +34,8 @@ if(FLANG_VENDOR)
     PROPERTIES COMPILE_DEFINITIONS "FLANG_VENDOR=\"${FLANG_VENDOR} \"")
 endif()
 
-
 add_flang_library(FortranCommon
+  CommandLineOpts.cpp
```

Comment:
It does not feel like it belongs to `FortranCommon`.  Can we put it into the same component library as `Pipelines.cpp`?

Reply:
Done